### ISSUE TITLE
TWBlue does not output to screen readers at times

### ISSUE THREAD

**Issue by manuelcortez** [manuelcortez/TWBlue#369]
I think you all have been hearing about this report. Just wanted to put it here so we can discuss about what happened. In short, sometimes, especially when apps built with different python versions are in the same machine, TWBlue is unable to output to screen readers, making the invisible interface to fail completely. Normally people deals with the problem by removing the folder %tmp%\gen_py, but we must fix this as clearly the code in fixes for win32com is not doing its job as it should.
Reports are [here](https://twitter.com/brianhartgen/status/1354715868937261056) and [here.](https://twitter.com/nikosdemetriou/status/1354724849294442497)

**Comment by beqabeqa473** [comment]
hello.

there is one more bug related to it.

when unpluggin default sound card, and sounds dissappear, twblue
doesn't output information to screenreader if tweet  has audio
indication or we are at the edge of buffer.


On 1/28/21, Manuel Cortéz <notifications@github.com> wrote:
> I think you all have been hearing about this report. Just wanted to put it
> here so we can discuss about what happened. In short, sometimes, especially
> when apps built with different python versions are in the same machine,
> TWBlue is unable to output to screen readers, making the invisible interface
> to fail completely. Normally people deals with the problem by removing the
> folder %tmp%\gen_py, but we must fix this as clearly the code in fixes for
> win32com is not doing its job as it should.
>
> --
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/manuelcortez/TWBlue/issues/369


-- 
with best regards Beqa Gozalishvili
Tell: +995593454005
Email: beqaprogger@gmail.com
Web: https://gozaltech.org
Skype: beqabeqa473
Telegram: https://t.me/gozaltech
facebook: https://facebook.com/gozaltech
twitter: https://twitter.com/beqabeqa473
Instagram: https://instagram.com/beqa.gozalishvili

**Comment by lukaszgo1** [comment]
Has it been considered to drop AO2 and replace it with [cytolk](https://github.com/pauliyobo/cytolk)?

**Comment by beqabeqa473** [comment]
if some time, twblue will be cross platform, this will lead problems

On 1/29/21, Łukasz Golonka <notifications@github.com> wrote:
> Has it been considered to drop AO2 and replace it with
> [cytolk](https://github.com/pauliyobo/cytolk)?
>
> --
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/manuelcortez/TWBlue/issues/369#issuecomment-769725974


-- 
with best regards Beqa Gozalishvili
Tell: +995593454005
Email: beqaprogger@gmail.com
Web: https://gozaltech.org
Skype: beqabeqa473
Telegram: https://t.me/gozaltech
facebook: https://facebook.com/gozaltech
twitter: https://twitter.com/beqabeqa473
Instagram: https://instagram.com/beqa.gozalishvili

**Comment by lukaszgo1** [comment]
The issue with AO2 manifests only under Windows AFAIK so we can always use AO2 for *nix and Mac and use Cytolk for Windows.
